ag/ag.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+//TODO(meling) workaround for Go module system
+import "gogoproto/gogo.proto";
```

Comment:
And this also causes no problem?

Reply:
Well, somehow my VSCode does not really get the idea of imports in `.proto` files and throws the same error for both paths, while the compiler has no problems with either. So, basically, yes, no problems.